server/scripts/mint-distribute.ts

```diff
@@ -0,0 +1,262 @@
+  ) => {
+    const myAddress = await getAddress(prvKey);
+    const txRes = await createAndMintFungibleGovernanceToken(gapi)(prvKey)(
+      tkName,
+      tkSymbol,
+      myAddress,
+      supply,
+    );
+    await waitUntilSuccessfull(gapi.transaction)(txRes.id);
+    const txDetails = await getTransactionDetails(gapi.transaction)(txRes.id);
+    const newResourceAddress = (
+      txDetails.transaction.receipt?.output as any[]
+    )[1].programmatic_json.fields[0].value;
+    // distribute
+    return distributeGovTokens(gapi)(prvKey)(
+      newResourceAddress,
+      supply,
+      receivers,
+    );
+  };
+
+
+const createAndMintFungibleGovernanceToken =
+  (gapi: GatewayApiClient) =>
+  (prvKey: PrivateKey) =>
+  async (
+    tkName: string,
+    tkSymbol: string,
+    receiver: string,
+    supply: number,
+  ) => {
+    const resourceManagerPkgAddr =
+      await RadixEngineToolkit.Utils.knownAddresses(NetworkId.Zabanet).then(
+        (x) => x.packageAddresses.resourcePackage,
+      );
+    const manifest = new ManifestBuilder()
+      // schema from https://github.com/radixdlt/radixdlt-scrypto/blob/7eaf1ef0db106236eb050dcd47245bfdcf3c739f/radix-engine/src/blueprints/resource/fungible/fungible_resource_manager.rs#L471
+      .callMethod(await faucetAddr(), "lock_fee", [decimal(25)])
+      .callFunction(
+        resourceManagerPkgAddr,
+        "FungibleResourceManager",
+        "create_with_initial_supply",
+        [
+          enumeration(0), // ownerrole
+          bool(true),
+          u8(0),
+          decimal(supply),
+          tuple(
+            enumeration(
+              1,
+              tuple(
+                enumeration(0), // minter (default is owner)
+                enumeration(0), // mint updater defaults to denayall when none
+              ),
+            ), // mint roles some(owner)
+            enumeration(0), // burn roles
+            enumeration(0), // freeze roles
+            enumeration(0), // recall roles
+            enumeration(0), // withdraw roles
+            enumeration(0), // deposit roles
+          ),
+          tuple(
+            map(
+              // metadata
+              ValueKind.String,
+              ValueKind.Tuple,
+              [
+                str("name"),
+                tuple(enumeration(1, enumeration(0, str(tkName))), bool(true)),
+              ],
+              [
+                str("symbol"),
+                tuple(
+                  enumeration(1, enumeration(0, str(tkSymbol))),
+                  bool(true),
+                ),
+              ],
+              [
+                str("description"),
+                tuple(
+                  enumeration(1, enumeration(0, str("test description"))),
+                  bool(true),
+                ),
+              ],
+            ),
+            map(
+              // metadata roles
+              ValueKind.String,
+              ValueKind.Enum,
+              // [str("metadata_setter"), enumeration(0)],
```

Comment:
Maybe make some comment about why it is commented out, or delete it if not needed.